code-challenges.test.js

```diff
@@ -85,22 +87,22 @@ const object3 = { number: -7 }
 
 
 
-// describe("isDivisible",() => {
-//     it("decides if the number inside it is evenly divisible by three or not", () => {
-//         expect(isDivisible(object1.number)).toEqual ("15 is divisible by three")
-//         expect(isDivisible(object2.number)).toEqual ("0 is divisible by three")
-//         expect(isDivisible(object3.number)).toEqual ("-7 is not divisible by three")
-//     })
-// })
+describe("isDivisible",() => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+        expect(isDivisible(object1.number)).toEqual ("15 is divisible by three")
+        expect(isDivisible(object2.number)).toEqual ("0 is divisible by three")
+        expect(isDivisible(object3.number)).toEqual ("-7 is not divisible by three")
+    })
+})
```

Comment:
Since the prompt says that the function takes an object, the argument should be an object, not the value inside the object